packages/web-components/case-card/src/element.ts

```diff
@@ -1,26 +1,16 @@
-/*
-Temporary utrecht css import.
-It should be taken from node_modules/@utrecht/paragraph-css/dist/index.mjs,
-but the module resolution is not working yet
- */
-import css, { utrechtParagraphCss, denhaagIconCss } from './css.js';
+import cardCss from '@gemeente-denhaag/card/dist/css.mjs';
 import { escapeXML } from './util.js';
 
 export interface CardData {
   lang?: string;
   linkLabel: string;
   dateTime?: string;
   href: string;
+  inactive?: boolean;
```

Comment:
`appearance="inactive"`? of misschien wel `appearance="archive"` omdat er geen `active` is, en als er wel `active` zou zijn is het verwarrend met `:active`.

Heeft wat voordelen, imho:
- je gelijk een API waarmee je uitbreidingen kan doen
- het is geen negation (inactive voelt een beetje als `notActive`, en `enabled` zou je ook niet als `notDisabled` willen maken.
- en je hoeft niet elke appearance * zowel inactive als active state te doen, scheelt complexiteit

Die laatste is natuurlijk ook beperkend, maar misschien wel goede zelfbescherming?

Anyway, ik ben benieuwd hoe jij hier over denkt!

Reply:
mee eens